monitor_internet_connection.py

```diff
@@ -149,12 +147,16 @@ def signal_handler(signal_received, frame):
     # Display exit message to console and record in log file.
     exit_time = datetime.datetime.now()
     exit_msg = "Monitoring Internet Connection stopped at : " + exit_time.strftime("%Y-%m-%d %H:%M:%S")
+    exit_csv = "STOPPED," + exit_time.strftime("%Y-%m-%d %H:%M:%S")
     print(exit_msg)
 
-    if enable_logfile:
+    if log_type == "text":
         with open(file, 'a') as writer:
             writer.write(exit_msg + "\n")
-
+    if log_type == "csv":
```

Comment:
elif would be better instead of multiple if statements since we need to check which log_type was chosen